.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,27 +1,28 @@
-## Summary
 
-*Short description*
+# Checklist
 
-## TODO
+- [ ] Target milestone
+- [ ] New tests
+- [ ] Updated documentation
+- [ ] External dependency updates
```

Comment:
Could this be made more understandable? Like "Python and Node.js dependencies updated?"